### DISCUSSION TITLE
how to increase the width of the chat window and PLACEHOLDER_AGENT_GOAL textarea?

### DISCUSSION THREAD

**Discussion by jerryho-quanta** [reworkd/AgentGPT#471] [Q&A]
Dear all,

I tried to remove the drawer  and it's ok...
How to increase the width of the chat window and PLACEHOLDER_AGENT_GOAL text area if disable the drawer?

Thanks,

**Comment by Rakly3** [comment]
In your browser, open the dev tools (F12)
select the line starting with `div id="layout" class="flex h-full w-full max-w-screen-xl flex-col items-center justify-between gap-1 `

in the 'Styles' section of the dev tools, look for `max-w-screen-xl` and disable it.
